### ISSUE TITLE
Feature parity with iTunes tags and compatibility fallback

### ISSUE THREAD

**Issue by theDanielJLewis** [Podcastindex-org/podcast-namespace#92]
I can't remember whether this was mentioned in the latest _Podcasting 2.0_, or if it was in Adam's appearance on _Buzzcast_, but I think it's worth discussing.

What do you think of replicating _all_ the features of the iTunes namespace?

The point for this would be to let app-developers support only a single namespace that has all the features, but then fallback to the iTunes tags if the matching Podcast tags aren't present.

`<itunes:episode>` would have an equivalent `<podcast:episode>` for the episode number. Similarly, `<itunes:title>` would have `<podcast:title>` for a podcast-app-specific title.

**Comment by daveajones** [comment]
Yes, that's definitely the long-term goal.  I think we talked about it Friday on the show.  That's where I'd like this thing to end up.  Just down the road a ways after we fry bigger fish.
